app/views/calculation/resident/properties/deductions/lettingsRelief.scala.html

```diff
@@ -48,16 +49,17 @@ <h1>@Messages("calc.resident.lettingsRelief.title")</h1>
 
     @form(action = controllers.routes.DeductionsController.submitLettingsRelief) {
 
+        <p>@helpText</p>
```

Comment:
Add class form-hint to line 37 in order to render the page as it existing before the change